dibs-easy-for-woocommerce.php

```diff
@@ -180,10 +180,13 @@ public function init() {
 				return;
 			}
 
-			if ( 'embedded' === $this->checkout_flow ) {
+			if ( in_array( $this->checkout_flow, array( 'embedded', 'inline' ), true ) ) {
```

Comment:
I believe you added a helper method for this. Could be good to use here too. As long as the functions file can be included before this that is.